Assets/BossRoom/Scripts/Client/AnimationCallbacks/AnimatorFootstepSounds.cs

```diff
@@ -48,6 +50,10 @@ public class AnimatorFootstepSounds : MonoBehaviour
         [Tooltip("Relative volume to play the clip at")]
         private float m_RunFootstepVolume = 1;
 
+        [SerializeField]
+        [Tooltip("At what point do we play \"running\" footsteps? Higher than this point, we don't play any footstep sounds")]
+        private float m_RunningPoint = 1.2f;
```

Comment:
Just thinking for less ambiguity we call this "...SpeedThreshold", or something of the sort for this and the other two speed 'points'.

Reply:
ok

Reply:
Just need to do this for the two fields below as well.

Reply:
oh I see what you mean

ok